game.js

```diff
@@ -1,11 +1,65 @@
-let trumps = document.querySelectorAll('.trump');
+/**
+ * Provides random number between min and max provided
+ *
+ * @param min
+ *          Minimum number required
+ * @param max
+ *          Maximum number required
+ * @returns {*}
+ *          Random number
+ */
+function randTime(min, max) {
+    return Math.random() * (max - min) + min;
+}
+
+// Available classes for trump position
+var positions = ['animateTop', 'animateLeft', 'animateRight'];
+
+// Available classes for trumps
+var trumps = ['.trump01','.trump02','.trump03','.trump04','.trump05','.trump06']
+
+/**
+ * Provides single random element from supplied array
+ *
+ * @param array
+ *             Source array
+ *
+ * @returns {*}
+ *             Random element
+ */
+function getRandom(array) {
+    var length = array.length;
```

Comment:
I'd like you all to provide a very good reason to use var instead of let! While this won't break your code, you should always opt for the safest option when scoping variables and preventing any weirdness.